include/iotc.h

```diff
@@ -328,26 +269,10 @@ extern iotc_state_t iotc_publish(iotc_context_handle_t iotc_h,
                                  void* user_data);
 
 /**
- * @brief Publish binary data to Cloud IoT Core on a given topic.
```

Comment:
Emphasis on the difference that here the PUBLISH payload may contain zero bytes. Compared to the iotc_publish.

Reply:
Done